tests/test_iotedgemodule.py

```diff
@@ -0,0 +1,38 @@
+import json
+import logging
+import os
+import pytest
+import shutil
+
+root_dir = os.getcwd()
+module_name = "iotedgeFilterModule"
+file_list = ["requirements.txt", "module.json", "main.py", "Dockerfile.arm32v7",
+             "Dockerfile.amd64.debug", "Dockerfile.amd64", ".gitignore"]
+image_repository = "localhost:5000/" + module_name
+
+
+@pytest.fixture(scope="module")
+def test_install_scaffold(request):
+    installmodulecommand = "cookiecutter --no-input " + root_dir + " module_name=" + \
```

Comment:
Wrap `root_dir` with `"` in case of spaces in `root_dir`

Reply:
updated